stack-cmbm/recipes/deploy-nginxapp.rb

```diff
@@ -0,0 +1,63 @@
+include_recipe 'nginx-app::server'
+include_recipe 'supervisor'
+
+# Under which `user` should supervisor start and run the `puma` process.
+user = if is_aws
+         'www-data'
```

Comment:
Better use a node attr here - either `node['nginx-app']['user']`, or something new in the stack attribute rb